### ISSUE TITLE
Custom FieldType - Can you set the default value for the Mandatory setting?

### ISSUE THREAD

**Issue by DaltonKirkCrafted** [umbraco/Umbraco.Forms.Issues#928]
I have made a custom field control and I would like it to be mandatory by default is this possible? 

Thanks

---
_This item has been added to our backlog [AB#33511](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/33511)_

**Comment by AndyButland** [comment]
I don't believe this is possible no, but it's a good idea so we'll keep this issue open and take it in as a feature request.

**Comment by umbracotrd** [comment]
Hi @DaltonKirkCrafted, we've added a new property in `Umbraco.Forms.Core.Interfaces.IFieldType` and `Umbraco.Forms.Core.FieldType`
```
/// <summary>
/// Gets or sets a value indicating whether the field should be mandatory by default or not.
/// </summary>
bool MandatoryByDefault { get; set; }
```
Developers who want to make their custom field type mandatory by default just need to implement/override that property. Note: Please make sure that the `SupportsMandatory` for that property is set to `true` (the default value for `SupportsMandatory` should be `true` if I'm not mistaken.
That property should be available in **v12.2.0**
